bfcpu-freq

```diff
@@ -72,16 +72,14 @@ if [ $bfversion == $BF1_PLATFORM_ID ]; then
 
 elif [ $bfversion == $BF2_PLATFORM_ID ]; then
 
-	padfreq=200 # MHz
-
-	if [ $is_livefish == true ]; then
+	if [ "$is_livefish" == "true" ]; then
```

Comment:
We never set $is_livefish, so this whole thing can be removed.

Reply:
done